blob/blob.go

```diff
@@ -366,8 +367,8 @@ func (b *Bucket) Attributes(ctx context.Context, key string) (Attributes, error)
 // is not found by the given key, which can be checked by calling IsNotExist.
```

Comment:
Docs (here and below) should explicitly state that `nil` options are treated the same as passing the zero value.